model/BUILD.bazel

```diff
@@ -0,0 +1,163 @@
+# TODO: Rename file to 'BUILD' post-Gradle. See https://github.com/oppia/oppia-android/issues/1532.
+'''In Bazel, proto files are built using the proto_library() and java_lite_proto_library() rules.
+The proto_library() rule creates a proto file library to be used in multiple languages.
+The java_lite_proto_library() rule takes in a proto_library target and generates java code.
+'''
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@rules_java//java:defs.bzl", "java_lite_proto_library")
+load("//model:src/main/proto/format_import_proto_library.bzl", "format_import_proto_library")
+
+proto_library(
```

Comment:
Just a thought, have a look here - https://github.com/google/startup-os/blob/master/WORKSPACE#L173 if this can be an optimisation for our work!!

Reply:
Interesting! I tried it out and it does work replace the `http_archive()` for `rules_proto` and `rules_java` with 
```
bind(
    name = "proto_compiler",
    actual = "//tools:protoc"
)

bind(
    name = "proto_java_toolchain",
    actual = "//tools:java_toolchain"
) 
```
then we can also omit the corresponding load statements from the model BUILD file. What do you think @BenHenning ?